doc/guide/installation.rst

```diff
@@ -16,6 +17,11 @@ The easiest way to keep your suite updated is to use
     .. code-block:: js
 
         {
+            "require": {
+                "behat/behat": "~3.0",
+                "behat/mink-extension": "~2.0@dev",
+                "behat/mink-goutte-driver": "*"
```

Comment:
I find it belongs to mink docs how to configure it. I wouldn't like to recommend installing goutte as not everyone need it.